index.html

```diff
@@ -36,5 +36,131 @@ <h1 class="heading"> Hey There.<br>I’m Ismail</h1>
+            <li><a href="#" alt="html">html</a></li>
+            <li><a href="#" alt="bootstrap">bootstrap</a></li>
+            <li><a href="#" alt="ruby">Ruby</a></li>
+        </ul>
+        <button class="see-btn ">See Project</button>
+    </section>
+    <section class="projects">
+        <h4 class="printing">Profesional Art Printing Data</h4>
+            <p class="print-info">A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-project">
+            <li><a href="#" alt="html">html</a></li>
+            <li><a href="#" alt="bootstrap">bootstrap</a></li>
+            <li><a href="#" alt="ruby">Ruby</a></li>
+        </ul>
+        <button class="see-btn ">See Project</button>
+    </section>
+    <section class="projects">
+        <h4 class="printing">Profesional Art Printing Data</h4>
+            <p class="print-info">A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-project">
+            <li><a href="#" alt="html">html</a></li>
+            <li><a href="#" alt="bootstrap">bootstrap</a></li>
+            <li><a href="#" alt="ruby">Ruby</a></li>
+        </ul>
+        <button class="see-btn ">See Project</button>
+    </section>
+    <section class="projects">
+        <h4 class="printing">Profesional Art Printing Data</h4>
+            <p class="print-info">A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-project">
+            <li><a href="#" alt="html">html</a></li>
+            <li><a href="#" alt="bootstrap">bootstrap</a></li>
+            <li><a href="#" alt="ruby">Ruby</a></li>
+        </ul>
+        <button class="see-btn ">See Project</button>
+    </section>
+    <section class="projects">
+        <h4 class="printing">Profesional Art Printing Data</h4>
+            <p class="print-info">A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-project">
+            <li><a href="#" alt="html">html</a></li>
+            <li><a href="#" alt="bootstrap">bootstrap</a></li>
+            <li><a href="#" alt="ruby">Ruby</a></li>
+        </ul>
+        <button class="see-btn ">See Project</button>
+    </section>
+    <section class="about-me">
+        <h5 class="about-heading">About Me</h5>
+        <p class="about-info"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <section class="about-section">
+            <buton class="see-btn resume">Get My Resume</buton>
```

Comment:
- [ ] For the header and the button kindly use the correct `font-family` as shown in Figma instead of `Tahoma` to match the design perfectly 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205096638-f97229a7-b459-43b9-8db4-32c33f985a51.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205096773-06381fa9-9c40-453c-80ff-5facd251e135.png)